### DISCUSSION TITLE
CefSharp.WPF.NetCore to support H.264?

### DISCUSSION THREAD

**Discussion by iexapl** [cefsharp/CefSharp#4525] [General]
How to use CefSharp.WPF.NetCore to support H.264?

**Comment by BuslikDrev** [comment]
https://github.com/cefsharp/CefSharp/wiki/General-Usage#multimedia-audiovideo
Используйте webm и не парьтесь. Качественный webm можно получить в программе "Format Factory".

**Comment by amaitland** [comment]
Duplicate of #3316
